dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -736,18 +733,15 @@ public void Test_Required_Params_Multiple_Params_Descriptor_Non_Empty_Value_Miss
             //Secrets are destroyed for security every time. Making the form useless. They need to be re-generated every time.
             //We're sending only one parameter when the descriptor says there's another one mandatory.
             final Map<String, Input> inputParamMap = ImmutableMap.of(
-                    "param1", Input.newInputParam("any-value".toCharArray(),false)
-                    //Please Notice We're NOT sending param2
+                    "param1", newInputParam("any-value".toCharArray(),false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "any-value" appears 5 times in this file; the first occurrence is on line 736](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)